package.json

```
@@ -10,8 +10,15 @@
     "test": "NODE_ENV=test jest --runInBand --detectOpenHandles --coverage",
     "kill-run": "npm run kill && npm run dev",
     "start": "NODE_ENV=production node ./server/index.js",
-    "heroku-postbuild": "NODE_ENV=production cd client && npm install && npm run build"
+    "heroku-postbuild": "NODE_ENV=production cd client && npm install && npm run build",
+    "lint-fix": "eslint server/ --fix",
+    "lint": "eslint server/",
+    "lint-react": "cd client && npm run lint"
   },
+  "pre-commit": [
+    "test",
+    "lint-fix"
+  ],
```

and add lint react to precommit,
```suggestion
 "pre-commit": [
    "test",
    "lint-fix",
    "lint-react-fix"
  ],
```

---

Done